lms/static/js/learner_dashboard/views/entitlement_unenroll_view.js

```diff
@@ -0,0 +1,157 @@
+(function(define) {
+    'use strict';
+    define(['backbone',
+        'jquery',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils'
+    ],
+        function(Backbone, $, gettext, HtmlUtils) {
+            return Backbone.View.extend({
+                el: '#unenroll-entitlement-modal',
+
+                triggerSelector: '.js-action-unenroll-entitlement',
+
+                submitButtonSelector: '#unenroll-entitlement-submit',
+
+                trackInfoSelector: '#entitlement-track-info',
```

Comment:
For all of these, convert them to classes. It's preferred to use classes over elements unless forced to by an outside constraint.

Reply:
Changed.